<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>messagewindow</class>
 <widget class="QWidget" name="messagewindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="exit">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>410</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文行楷</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendmessage">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>410</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文行楷</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>发送消息</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>40</y>
     <width>271</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文行楷</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>我收到的消息</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>41</y>
     <width>256</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文行楷</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>我发送的消息</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>240</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文行楷</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>回复消息</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>260</y>
     <width>271</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="backmessage">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>410</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文行楷</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>回复</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exit</sender>
   <signal>clicked()</signal>
   <receiver>messagewindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>497</x>
     <y>401</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
